mmv1/third_party/terraform/.teamcity/components/builds/build_triggers.kt

```diff
@@ -36,7 +36,8 @@ class NightlyTriggerConfiguration(
 fun Triggers.runNightly(config: NightlyTriggerConfiguration) {
 
     schedule{
-        enabled = config.nightlyTestsEnabled
+        // enabled = config.nightlyTestsEnabled
+        enabled = false
```

Comment:
I'm pulling down the test deployment, so reverting this now

```suggestion
        enabled = config.nightlyTestsEnabled
```